src/driver/mock.rs

```diff
@@ -157,19 +170,22 @@ impl MockDatabaseConnection {
 
     /// Create a statement block  of SQL statements that execute together.
     #[instrument(level = "trace")]
-    pub fn begin(&self) {
-        self.mocker.lock().unwrap().begin()
+    pub fn begin(&self) -> Result<(), DbErr> {
+        self.mocker.lock().map_err(exec_err)?.begin();
```

Comment:
Returning an Error here does not mean much as the only possibility that a lock fail is another thread already panicked

Reply:
https://github.com/SeaQL/sea-orm/pull/1231/commits/14903a2e6ce435f33e197435d81203381e1a81e9